### DISCUSSION TITLE
Equivalent of docker run?

### DISCUSSION THREAD

**Discussion by dmurat** [testcontainers/testcontainers-node#696] [Q&A] [answered]
I would like to run a container in a similar way to `docker run`. As I understand, for this purpose, in Testcontainers for Java, there is [One shot startup strategy](https://java.testcontainers.org/features/startup_and_waits/#one-shot-startup-strategy-example) available. Is there something similar in Testcontainers for Node?

**Comment by cristianrgreco** [comment, marked as answer]
Hi @dmurat. That's interesting I haven't heard of the One shot startup strategy being referred to as a `docker run` equivalent before, but I see why it is. This startup strategy doesn't yet exist in testcontainers-node. 

However you can achieve the same outcome quite easily:

```js
const container = await new GenericContainer("alpine")
  .withCommand("echo hello world")
  .start()
```

In this case the container will run the command and exit.

**Reply by dmurat** [reply]
Thank you. This is very helpful for me.
